web-registry/src/lib/ecocredit/ecocredit.config.ts

```diff
@@ -0,0 +1,14 @@
+// This is a batch denom mapping between v1(left) and v1Alplha(right) for mainnet.
+// This mapping has been created to find a transaction hashes made with v1Aplha.
+// It is needed because logs produce with v1Alpha have a different batch denom pattern from v1.
+// Also a v1Alpha batch denom can't be deduce from a v1 one, hence the static mapping.
```

Comment:
best to stick to the exact proto version name
```suggestion
// It is needed because logs produce with v1alpha1 have a different batch denom pattern from v1.
// Also a v1alpha1 batch denom can't be deduced from a v1 one, hence the static mapping.
```